src/components/Navbar.tsx

```diff
@@ -1,7 +1,12 @@
-import React from "react";
+import React,{useState} from "react";
 import { GymName } from "../constants";
-
 const Navbar = () => {
+// set the default dropmenu as false because we want to hide it in smaller screen
+const[dropMenu, setDropMenu] = useState(false) 
+// function for toggling drop menu
+const toggleDropMenu=()=>{
+setDropMenu(!dropMenu)
+}
```

Comment:
Please, add proper spacing for the changes. 
If you do not want to worry about this in the future, I recommend you to use https://prettier.io/